cohort-engine-api/src/main/java/com/ibm/cohort/engine/api/service/model/EnhancedHealthCheckResults.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * (C) Copyright IBM Corp. 2022, 2022
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.cohort.engine.api.service.model;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import com.ibm.cohort.annotations.Generated;
+
+import io.swagger.annotations.ApiModel;
+
+@Generated
+@ApiModel(value="EnhancedHealthCheckResults", description="An object containing the results of an attempt to connect to the FHIR data server and (optional) terminology server")
+public class EnhancedHealthCheckResults {
+
+	@NotNull
+	@Valid
+	private FhirServerConnectionStatusInfo dataServerConnectionResults;
+	private FhirServerConnectionStatusInfo terminologyServerConnectionResults;
```

Comment:
It visualizes in swagger a bit better with individual variables for each server type. Adding a list then adds a bunch of list processing checks like making sure you only have one of each type in the list, iterating through the list to find the correct type, handling errors for if someone puts in too many into the list etc, and I thought it was cleaner/simpler to just have the 2. If we end up adding another connection point, then I would think it would be worth it to do the list.